app/code/Magento/EavGraphQl/README.md

```diff
@@ -1,3 +1,15 @@
-# EavGraphQl
+# Magento_EavGraphQl module
 
-**EavGraphQl** primarily provides the GraphQl module information to generate metadata for Eav attributes.
+Magento_EavGraphQl module extends Magento_GraphQl and Magento_Eav modules to provides type and resolver information for  GraphQL API.
```

Comment:
Please fix double space before the "GraphQL" word and change "to provides" either to "and provides" or "to provide".